.github/workflows/make-release.yaml

```diff
@@ -88,83 +72,32 @@ jobs:
         with:
           path: pkg
 
-      - name: Show directory
+      - name: Create archive of random library
```

Comment:
I read this like 5 times in confusion before realising that it meant build dust's random library, as opposed to  building any library _at_ random :laughing: